### PR TITLE
Fix #2432: fixed about us landscape orientation ui

### LINKED ISSUE #2432: About us screen is hadly accessible in landscape mode
**Summary:**

- When you change the orientation of about us fragment to landscape mode , the links are no longer visible on the screen. You can scroll through the items present in about us fragment but their visible range is very limited

**Steps to reproduce:**

- open about us from side navigation

**Expected behaviour:**

- items present in about us fragment should been visible

**Observed behaviour:**

WhatsApp Image 2023-11-04 at 11 31 45 AM

**Device and Android version:**

Device: Pixel 7
Android: 14

### PR DESCRIPTION
**Pull request by ayushkeshri19**
https://github.com/openMF/mifos-mobile/assets/90385832/5e01a069-75b7-4c03-8625-a97be0af65e2

Fixes #2432

Please Add Screenshots If there are any UI changes.

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: app/src/main/java/org/mifos/mobile/ui/about/AboutUsScreen.kt [hunk 1]
```diff
@@ -17,49 +17,57 @@ import java.util.*
 @SuppressWarnings("UnusedMaterial3ScaffoldPaddingParameter")
 @Composable
 fun AboutUsScreen(viewModel: AboutUsViewModel) {
-    Column(
+
+
```
Comment (PratyushSingh07) [inline app/src/main/java/org/mifos/mobile/ui/about/AboutUsScreen.kt]: remove these extra spaces

File: app/src/main/java/org/mifos/mobile/ui/about/AboutUsFragment.kt [hunk 2]
```diff
@@ -30,6 +31,7 @@ class AboutUsFragment : BaseFragment() {
 
     private val viewModel: AboutUsViewModel by viewModels()
 
+    @SuppressLint("UnusedMaterial3ScaffoldPaddingParameter")
```
Comment (PratyushSingh07) [inline app/src/main/java/org/mifos/mobile/ui/about/AboutUsFragment.kt:34]: no need to add this , you can remove scaffold from this fragment

**Review by PratyushSingh07 — CHANGES_REQUESTED** [review]

**Comment by PratyushSingh07** [comment]
please squash your commits into one . You can follow the guide [here](https://www.git-tower.com/learn/git/faq/git-squash)

**Comment by neeldoshii** [comment]
@PratyushSingh07 can I work on this and complete the PR so the issue can be fixed and PR be closed. This issue and PR has been staled for a month can I work on it?

**Comment by PratyushSingh07** [comment]
This PR is complete @neeldoshii

**Pull request merged**